src/status_im/contexts/wallet/events/accounts.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im.contexts.wallet.events.accounts
```

Comment:
Added new namespace because this event is related to accounts, I didn't move the others that are related because it'd add a lot of noise to this PR.